cql3/authorized_prepared_statements_cache.hh

```diff
@@ -59,7 +59,8 @@ public:
     bool operator==(const authorized_prepared_statements_cache_key&) const = default;
 
     static size_t hash(const auth::authenticated_user& user, const cql3::prepared_cache_key_type::cache_key_type& prep_cache_key) {
-        return utils::hash_combine(std::hash<auth::authenticated_user>()(user), utils::tuple_hash()(prep_cache_key));
+        return utils::hash_combine(std::hash<auth::authenticated_user>()(user),
```

Comment:
Is this change unrelated?

Reply:
it is related. the underlying type of `cql3::prepared_cache_key_type::cache_key_type` is changed, it is not a `std::pair<>` anymore after removing the `THRIFT_PREP_ID` component from it. see also the change in `cql3/prepared_statements_cache.hh`.